### ISSUE TITLE
Unable to run 'lein test' without errors.

### ISSUE THREAD

**Issue by LudoTheHUN** [uncomplicate/clojurecl#2]
Also
`lein repl`
and then just

```
(ns example
  (:use [uncomplicate.clojurecl core info]))
```

throws:

```
CompilerException java.lang.RuntimeException: Unable to find static field: BYTES in class java.lang.Float, compiling:(uncomplicate/clojurecl/core.clj:733:5)
```

Here the full lein test run output 

```
~/code/lg-clones/clojurecl $ lein test
OpenJDK 64-Bit Server VM warning: Failed to reserve shared memory (errno = 22).
OpenJDK 64-Bit Server VM warning: Failed to reserve shared memory (errno = 22).
Boxed math warning, /tmp/form-init1756028734995598981.clj:1:11175 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, /tmp/form-init1756028734995598981.clj:1:11254 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, midje/util/ecosystem.clj:22:9 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,long).
Boxed math warning, midje/util/ecosystem.clj:26:9 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,long).
Boxed math warning, midje/util/ecosystem.clj:30:9 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,long).
Reflection warning, midje/util/ecosystem.clj:48:12 - call to java.io.File ctor can't be resolved.
Boxed math warning, ordered/map.clj:75:45 - call: public static long clojure.lang.Numbers.xor(java.lang.Object,java.lang.Object).
Boxed math warning, ordered/map.clj:75:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, swiss/arrows.clj:16:6 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,long).
Reflection warning, midje/clojure/core.clj:91:17 - reference to field hasRoot can't be resolved.
Reflection warning, midje/util/pile.clj:109:9 - call to method contains can't be resolved (target class is unknown).
Reflection warning, midje/config.clj:19:32 - call to method contains can't be resolved (target class is unknown).
Reflection warning, midje/emission/clojure_test_facade.clj:26:19 - reference to field toString can't be resolved.
Boxed math warning, ordered/set.clj:74:15 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Reflection warning, midje/emission/plugins/util.clj:101:8 - reference to field getClass can't be resolved.
Reflection warning, midje/emission/plugins/util.clj:101:8 - reference to field getName can't be resolved.
Boxed math warning, flare/util.clj:17:9 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,long).
Boxed math warning, flare/sequential.clj:9:45 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(long,java.lang.Object).
Boxed math warning, flare/sequential.clj:10:33 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(long,java.lang.Object).
Reflection warning, flare/string.clj:23:24 - reference to field operation can't be resolved.
Reflection warning, flare/string.clj:23:43 - reference to field text can't be resolved.
Boxed math warning, flare/string.clj:97:8 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(long,java.lang.Object).
Boxed math warning, flare/string.clj:97:5 - call: public static java.lang.Number clojure.lang.Numbers.divide(java.lang.Object,long).
Boxed math warning, flare/string.clj:141:8 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, flare/string.clj:175:9 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_multiply(java.lang.Object,long).
Boxed math warning, midje/emission/plugins/default_failure_lines.clj:81:46 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:51:29 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:51:22 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:54:22 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:61:19 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:65:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, midje/emission/plugins/default.clj:65:35 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, midje/parsing/util/zip.clj:47:7 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/parsing/util/zip.clj:49:12 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, midje/parsing/util/file_position.clj:33:54 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Reflection warning, midje/parsing/util/file_position.clj:59:5 - reference to field getFileName can't be resolved.
Reflection warning, midje/data/metaconstant.clj:43:5 - reference to field getNamespace on java.lang.Object can't be resolved.
Boxed math warning, midje/parsing/util/error_handling.clj:12:44 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, midje/parsing/util/error_handling.clj:40:30 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Reflection warning, midje/checking/checkers/chatty.clj:60:20 - call to java.lang.Error ctor can't be resolved.
Boxed math warning, midje/checking/checkers/simple.clj:38:15 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:40:13 - call: public static java.lang.Number clojure.lang.Numbers.minusP(java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:49:28 - call: public static java.lang.Number clojure.lang.Numbers.minusP(java.lang.Object,java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:49:15 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:50:28 - call: public static java.lang.Number clojure.lang.Numbers.addP(java.lang.Object,java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:50:15 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, midje/checking/checkers/simple.clj:52:31 - call: public static double clojure.lang.Numbers.multiplyP(double,java.lang.Object).
Reflection warning, midje/checking/checkers/simple.clj:84:24 - reference to field throwable can't be resolved.
Boxed math warning, clojure/math/combinatorics.clj:105:55 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:67 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:64 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:55 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:55 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:67 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:64 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:55 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:105:45 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:108:17 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:109:34 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:110:21 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:110:36 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:113:39 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:113:47 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:113:60 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:116:37 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:126:9 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:127:36 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:129:13 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:131:47 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:132:18 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:133:18 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:139:7 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:141:35 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:143:22 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:143:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:144:16 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:154:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:154:58 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:156:26 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:156:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:156:73 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:156:13 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:157:27 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:157:58 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:191:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:193:15 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:248:19 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:253:21 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:254:44 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:255:15 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:256:46 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:310:28 - call: public static boolean clojure.lang.Numbers.isNeg(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:312:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:312:43 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
combinatorics.clj:312 recur arg for primitive local: acc is not matching primitive, had: Object, needed: long
Auto-boxing loop arg: acc
Boxed math warning, clojure/math/combinatorics.clj:312:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:312:43 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:319:28 - call: public static boolean clojure.lang.Numbers.isNeg(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:321:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:323:15 - call: public static java.lang.Number clojure.lang.Numbers.quotient(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:323:35 - call: public static java.lang.Number clojure.lang.Numbers.remainder(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:328:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:329:44 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:335:24 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:369:69 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:369:69 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:384:14 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:385:14 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:386:14 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:405:9 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:416:24 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:453:5 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:477:5 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:478:5 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:479:5 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:481:10 - call: public static java.lang.Number clojure.lang.Numbers.quotient(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:481:5 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:481:31 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:482:38 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:482:33 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:482:47 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:483:35 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:482:11 - call: public static java.lang.Number clojure.lang.Numbers.divide(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:489:7 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:492:7 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:497:62 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:516:26 - call: public static java.lang.Number clojure.lang.Numbers.quotient(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:519:8 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
combinatorics.clj:520 recur arg for primitive local: y is not matching primitive, had: Object, needed: long
Auto-boxing loop arg: y
Boxed math warning, clojure/math/combinatorics.clj:516:26 - call: public static java.lang.Number clojure.lang.Numbers.quotient(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:519:8 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:545:13 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:546:49 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:547:13 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:548:57 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:549:38 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:559:13 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:563:71 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:564:13 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:565:55 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:566:50 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:571:24 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:587:24 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:593:11 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:595:27 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:614:23 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_multiply(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:614:14 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(long,java.lang.Object).
combinatorics.clj:613 recur arg for primitive local: index is not matching primitive, had: Object, needed: long
Auto-boxing loop arg: index
Boxed math warning, clojure/math/combinatorics.clj:614:23 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_multiply(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:614:14 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
combinatorics.clj:621 recur arg for primitive local: index is not matching primitive, had: Object, needed: long
Auto-boxing loop arg: index
Boxed math warning, clojure/math/combinatorics.clj:665:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:665:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:665:19 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:665:12 - call: public static java.lang.Object clojure.lang.Numbers.max(long,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:665:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:665:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:665:19 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:665:12 - call: public static java.lang.Object clojure.lang.Numbers.max(long,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:664:27 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:678:22 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:679:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:679:28 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:680:45 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:681:32 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:682:34 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:684:45 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:684:36 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:687:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:687:67 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:687:63 - call: public static boolean clojure.lang.Numbers.equiv(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:687:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:687:88 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:687:36 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:690:34 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:691:21 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:696:32 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:695:29 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:700:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:702:65 - call: public static boolean clojure.lang.Numbers.equiv(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:701:50 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:704:35 - call: public static boolean clojure.lang.Numbers.equiv(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:706:50 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:706:47 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:706:42 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:707:52 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:707:49 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:710:44 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:711:61 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:711:44 - call: public static java.lang.Object clojure.lang.Numbers.max(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:715:48 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:725:30 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:726:26 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:804:29 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:806:46 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:808:38 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:812:52 - call: public static java.lang.Object clojure.lang.Numbers.min(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:813:41 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:814:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:815:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:820:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:821:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:826:19 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:827:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:829:19 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:830:22 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:830:19 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:831:14 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:831:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:832:39 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:836:56 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:836:48 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:837:69 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:836:56 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:836:48 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:837:69 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:834:46 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:845:20 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:848:22 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:852:18 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:852:30 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:852:15 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_multiply(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:852:12 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:858:44 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:858:44 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:857:59 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:859:38 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:866:56 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:866:51 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:867:46 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:867:39 - call: public static java.lang.Object clojure.lang.Numbers.max(long,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:870:34 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:873:31 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:874:36 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:876:36 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:877:42 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:884:13 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:900:30 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:901:26 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,long).
Boxed math warning, clojure/math/combinatorics.clj:906:59 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:906:59 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:910:75 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:910:75 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:910:75 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, clojure/math/combinatorics.clj:910:75 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_dec(java.lang.Object).
Boxed math warning, midje/emission/boundaries.clj:15:11 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/emission/boundaries.clj:22:14 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, midje/emission/boundaries.clj:22:14 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, midje/checking/checkables.clj:83:23 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, midje/checking/checkables.clj:89:8 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Reflection warning, midje/data/compendium.clj:60:38 - call to method indexOf can't be resolved (target class is unknown).
Boxed math warning, midje/data/compendium.clj:61:32 - call: public static boolean clojure.lang.Numbers.isNeg(java.lang.Object).
Boxed math warning, midje/data/compendium.clj:63:40 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, midje/parsing/1_to_explicit_form/facts.clj:183:3 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, midje/parsing/0_to_fact_form/tabular.clj:56:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(long,java.lang.Object).
Boxed math warning, midje/parsing/0_to_fact_form/formulas.clj:22:9 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, midje/parsing/0_to_fact_form/formulas.clj:69:21 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, potemkin/utils.clj:14:47 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, potemkin/utils.clj:15:43 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, byte_streams/char_sequence.clj:37:21 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,long).
Boxed math warning, byte_streams/char_sequence.clj:43:19 - call: public static double clojure.lang.Numbers.divide(java.lang.Object,double).
Boxed math warning, byte_streams.clj:546:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, byte_streams.clj:546:32 - call: public static java.lang.Object clojure.lang.Numbers.min(long,java.lang.Object).
Boxed math warning, byte_streams.clj:653:29 - call: public static boolean clojure.lang.Numbers.lte(long,java.lang.Object).
Boxed math warning, byte_streams.clj:654:37 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(long,java.lang.Object).
Boxed math warning, byte_streams.clj:662:29 - call: public static java.lang.Object clojure.lang.Numbers.min(java.lang.Object,java.lang.Object).
Boxed math warning, byte_streams.clj:663:36 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, byte_streams.clj:761:15 - call: public static java.lang.Object clojure.lang.Numbers.min(long,java.lang.Object).
Boxed math warning, byte_streams.clj:764:39 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(long,java.lang.Object).
Boxed math warning, byte_streams.clj:767:25 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,long).
Boxed math warning, clojure/core/async/impl/channels.clj:117:41 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async/impl/channels.clj:200:22 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async/impl/timers.clj:48:23 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,long).
Boxed math warning, clojure/tools/analyzer/utils.clj:145:20 - call: public static boolean clojure.lang.Numbers.gte(java.lang.Object,long).
Boxed math warning, clojure/tools/analyzer.clj:634:29 - call: public static boolean clojure.lang.Numbers.lte(java.lang.Object,java.lang.Object).
Reflection warning, clojure/data/priority_map.clj:215:19 - call to method equiv on java.lang.Object can't be resolved (no such method).
Boxed math warning, clojure/core/cache.clj:146:38 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:150:30 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/core/cache.clj:174:28 - call: public static boolean clojure.lang.Numbers.gte(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:200:23 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,long).
Boxed math warning, clojure/core/cache.clj:200:61 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:213:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:219:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:220:11 - call: public static boolean clojure.lang.Numbers.gte(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:237:20 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:251:33 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:251:30 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:265:27 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:266:10 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:266:7 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:301:9 - call: public static boolean clojure.lang.Numbers.gte(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:421:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:444:17 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:445:11 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:475:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:476:41 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_minus(java.lang.Object).
Boxed math warning, clojure/core/cache.clj:575:30 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:577:11 - call: public static boolean clojure.lang.Numbers.equiv(java.lang.Object,long).
Boxed math warning, clojure/core/cache.clj:587:30 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:598:24 - call: public static boolean clojure.lang.Numbers.lte(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:608:30 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:618:36 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/cache.clj:619:36 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Reflection warning, clojure/core/memoize.clj:72:23 - reference to field cache can't be resolved.
Boxed math warning, clojure/core/async/impl/ioc_macros.clj:837:33 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async/impl/ioc_macros.clj:887:25 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,long).
Boxed math warning, clojure/core/async/impl/ioc_macros.clj:888:24 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_add(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:443:14 - call: public static boolean clojure.lang.Numbers.isPos(java.lang.Object).
Boxed math warning, clojure/core/async.clj:578:5 - call: public static java.lang.Object clojure.lang.Numbers.min(java.lang.Object,long).
Boxed math warning, clojure/core/async.clj:580:18 - call: public static boolean clojure.lang.Numbers.lt(long,java.lang.Object).
Boxed math warning, clojure/core/async.clj:639:28 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/core/async.clj:641:5 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:641:5 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:641:5 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:641:5 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:831:8 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:831:8 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:883:34 - call: public static boolean clojure.lang.Numbers.isZero(java.lang.Object).
Boxed math warning, clojure/core/async.clj:886:8 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:886:8 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:936:20 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:940:27 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:1032:3 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:1032:3 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:1086:37 - call: public static java.lang.Number clojure.lang.Numbers.unchecked_inc(java.lang.Object).
Boxed math warning, clojure/core/async.clj:1087:29 - call: public static boolean clojure.lang.Numbers.lt(java.lang.Object,java.lang.Object).
Boxed math warning, clojure/core/async.clj:1091:29 - call: public static boolean clojure.lang.Numbers.gt(java.lang.Object,long).
Warning: *opencl-2* not declared dynamic and thus is not dynamically rebindable, but its name suggests otherwise. Please either indicate ^:dynamic *opencl-2* or change the name. (uncomplicate/clojurecl/core.clj:80)
Exception in thread "main" java.lang.RuntimeException: Unable to find static field: BYTES in class java.lang.Float, compiling:(uncomplicate/clojurecl/core.clj:733:5)
	at clojure.lang.Compiler.analyze(Compiler.java:6543)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$HostExpr$Parser.parse(Compiler.java:996)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6723)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6718)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:5861)
	at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5296)
	at clojure.lang.Compiler$FnExpr.parse(Compiler.java:3925)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6721)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6711)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$MapExpr.parse(Compiler.java:3050)
	at clojure.lang.Compiler.analyze(Compiler.java:6532)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$InvokeExpr.parse(Compiler.java:3791)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6725)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.analyze(Compiler.java:6485)
	at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:5861)
	at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5296)
	at clojure.lang.Compiler$FnExpr.parse(Compiler.java:3925)
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:6721)
	at clojure.lang.Compiler.analyze(Compiler.java:6524)
	at clojure.lang.Compiler.eval(Compiler.java:6779)
	at clojure.lang.Compiler.load(Compiler.java:7227)
	at clojure.lang.RT.loadResourceScript(RT.java:371)
	at clojure.lang.RT.loadResourceScript(RT.java:362)
	at clojure.lang.RT.load(RT.java:446)
	at clojure.lang.RT.load(RT.java:412)
	at clojure.core$load$fn__5448.invoke(core.clj:5866)
	at clojure.core$load.doInvoke(core.clj:5865)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invoke(core.clj:5671)
	at clojure.core$load_lib$fn__5397.invoke(core.clj:5711)
	at clojure.core$load_lib.doInvoke(core.clj:5710)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invoke(core.clj:632)
	at clojure.core$load_libs.doInvoke(core.clj:5753)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invoke(core.clj:632)
	at clojure.core$require.doInvoke(core.clj:5832)
	at clojure.lang.RestFn.invoke(RestFn.java:457)
	at uncomplicate.clojurecl.core_test$eval190$loading__5340__auto____191.invoke(core_test.clj:1)
	at uncomplicate.clojurecl.core_test$eval190.invoke(core_test.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6782)
	at clojure.lang.Compiler.eval(Compiler.java:6771)
	at clojure.lang.Compiler.load(Compiler.java:7227)
	at clojure.lang.RT.loadResourceScript(RT.java:371)
	at clojure.lang.RT.loadResourceScript(RT.java:362)
	at clojure.lang.RT.load(RT.java:446)
	at clojure.lang.RT.load(RT.java:412)
	at clojure.core$load$fn__5448.invoke(core.clj:5866)
	at clojure.core$load.doInvoke(core.clj:5865)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invoke(core.clj:5671)
	at clojure.core$load_lib$fn__5397.invoke(core.clj:5711)
	at clojure.core$load_lib.doInvoke(core.clj:5710)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invoke(core.clj:632)
	at clojure.core$load_libs.doInvoke(core.clj:5749)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invoke(core.clj:632)
	at clojure.core$require.doInvoke(core.clj:5832)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invoke(core.clj:632)
	at user$eval93.invoke(form-init1756028734995598981.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6782)
	at clojure.lang.Compiler.eval(Compiler.java:6772)
	at clojure.lang.Compiler.load(Compiler.java:7227)
	at clojure.lang.Compiler.loadFile(Compiler.java:7165)
	at clojure.main$load_script.invoke(main.clj:275)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invoke(main.clj:308)
	at clojure.main$null_opt.invoke(main.clj:343)
	at clojure.main$main.doInvoke(main.clj:421)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.lang.RuntimeException: Unable to find static field: BYTES in class java.lang.Float
	at clojure.lang.Util.runtimeException(Util.java:221)
	at clojure.lang.Compiler.analyzeSymbol(Compiler.java:6916)
	at clojure.lang.Compiler.analyze(Compiler.java:6506)
	... 85 more
Tests failed.
```

**Comment by LudoTheHUN** [comment]
I'm on ubuntu with an nvidia card...  Anyway, any ideas on how to address this welcome.

**Comment by LudoTheHUN** [comment]
Also
```
~/code/lg-clones/clojurecl $ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.5) (7u79-2.5.5-0ubuntu0.14.04.2)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
 ```

**Comment by LudoTheHUN** [comment]
As per the stack trace... this issue comes from

```
(extend-type (Class/forName "[F")
  Mem
  (ptr [this]
    (Pointer/to ^floats this))
  (size [this]
    (* Float/BYTES (alength ^floats this)))
  Argument
  (set-arg [this kernel n]
    (with-check (CL/clSetKernelArg kernel n
                                   (* Float/BYTES (alength ^floats this))
                                   (Pointer/to ^floats this))
```

Specifically `Float/BYTES`

When I remove the above expression, I get the same error, but now from Double/BYTES.

I don't see the BYTES method / static in any of the java specifications. Is this a new java8 feature?

**Comment by LudoTheHUN** [comment]
OK, that specific error is due to the Java 1.8 requirement. Please make that very clear in the readme.

**Comment by LudoTheHUN** [comment]
I've upgraded to jdk8.
Things still fail for me, now with `java.lang.UnsupportedOperationException: The function clReleaseDevice is not supported` (amongs many others).
I guess it's because I don't have opencl2 (on an Nvidia card), on ubuntu...  ( saw similar on windows 8.1)

Did you develop against an AMD card? On ubuntu?

**Comment by LudoTheHUN** [comment]
OK, I've made one more progress.

First I tried to follow https://devtalk.nvidia.com/default/topic/465288/problem-running-opencl-programs/  with no luck.
But working through
http://wiki.tiker.net/OpenCLHowTo#Installing_the_Nvidia_ICD
I installed opencl, maching my nvidia driver (which I bumped from v 304.125 to 352.21) with
`sudo apt-get install nvidia-libopencl1-352`

`lein test` now throws new set of errors at me:

```
java.lang.UnsatisfiedLinkError: no JOCL_0_2_0-linux-x86_64 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1865)
...
...big stack
...
Could not load libOpenCL.so, error libOpenCL.so: cannot open shared object file: No such file or directory
```

**Comment by LudoTheHUN** [comment]
Also installed:
`sudo apt-get install nvidia-opencl-icd-352`
and
`sudo apt-get install opencl-headers`

**Comment by LudoTheHUN** [comment]
Also, downloaded and installed the network installer here.
https://developer.nvidia.com/cuda-downloads?sid=898012
then as per:
http://developer.download.nvidia.com/compute/cuda/7_0/Prod/doc/CUDA_Getting_Started_Linux.pdf
I ran 
`sudo apt-get install cuda`
..which is now downloading 1.6GB of stuff.

**Comment by LudoTheHUN** [comment]
Installing cuda seems to have downgraded my nvidia drivers to version 346.

**Comment by LudoTheHUN** [comment]
Impressive... after rebooting , `lein test` now manages to get java to have a fatal error and core dump.

**Comment by LudoTheHUN** [comment]
I'll leave this for now. It seems I did not win the 'make opencl work Russian roulette' today.
Here the error now:

```
FAIL "num-devices tests." at (core_test.clj:33)
Actual result did not agree with the checking function.
        Actual result: 1
    Checking function: (throws ExceptionInfo)

FAIL "num-devices tests." at (core_test.clj:36)
    Expected: true
      Actual: false

FAIL "num-devices tests." at (core_test.clj:41)
    Expected: 1
      Actual: clojure.lang.ExceptionInfo: OpenCL error: CL_INVALID_DEVICE_TYPE. {:name "CL_INVALID_DEVICE_TYPE", :code -31, :type :opencl-error, :details nil}
              uncomplicate.clojurecl.utils$error.invokePrim(utils.clj:96)
              uncomplicate.clojurecl.utils$error.invoke(utils.clj:99)
              uncomplicate.clojurecl.core$num_devices_STAR_.invokePrim(core.clj:250)
              uncomplicate.clojurecl.core$num_devices.invokePrim(core.clj:280)
              uncomplicate.clojurecl.core_test$eval18190$fn__18191$fn__18192$fn__18205.invoke(core_test.clj:42)
              uncomplicate.clojurecl.core_test$eval18190$fn__18191$fn__18192.invoke(core_test.clj:28)
              uncomplicate.clojurecl.core_test$eval18190$fn__18191.invoke(core_test.clj:28)
              uncomplicate.clojurecl.core_test$eval18190.invoke(core_test.clj:28)

FAIL "num-devices tests." at (core_test.clj:50)
Actual result did not agree with the checking function.
        Actual result: 1
    Checking function: (throws ExceptionInfo)

FAIL "devices tests" at (core_test.clj:58)
Actual result did not agree with the checking function.
        Actual result: #object["[Lorg.jocl.cl_device_id;" 0x48b3ce9b "[Lorg.jocl.cl_device_id;@48b3ce9b"]
    Checking function: (throws ExceptionInfo)

FAIL "devices tests" at (core_test.clj:63)
    Expected: []
      Actual: clojure.lang.ExceptionInfo: OpenCL error: CL_INVALID_DEVICE_TYPE. {:name "CL_INVALID_DEVICE_TYPE", :code -31, :type :opencl-error, :details nil}
              uncomplicate.clojurecl.utils$error.invokePrim(utils.clj:96)
              uncomplicate.clojurecl.utils$error.invoke(utils.clj:99)
              uncomplicate.clojurecl.core$num_devices_STAR_.invokePrim(core.clj:250)
              uncomplicate.clojurecl.core$devices_STAR_.invokePrim(core.clj:308)
              uncomplicate.clojurecl.core$devices.doInvoke(core.clj:344)
              uncomplicate.clojurecl.core_test$eval18222$fn__18223$fn__18224$fn__18233.invoke(core_test.clj:63)
              uncomplicate.clojurecl.core_test$eval18222$fn__18223$fn__18224.invoke(core_test.clj:53)
              uncomplicate.clojurecl.core_test$eval18222$fn__18223.invoke(core_test.clj:53)
              uncomplicate.clojurecl.core_test$eval18222.invoke(core_test.clj:53)

FAIL "devices tests" at (core_test.clj:65)
    Expected: org.jocl.cl_device_id
      Actual: nil

FAIL "devices tests" at (core_test.clj:72)
Actual result did not agree with the checking function.
        Actual result: [#object[org.jocl.cl_device_id 0x6b7ff02a "cl_device_id[0x7fdcecfd69d0]"]]
    Checking function: (throws ExceptionInfo)

FAIL "Root level devices resource management." at (core_test.clj:81)
    Expected: 1
      Actual: clojure.lang.ExceptionInfo: OpenCL error: CL_INVALID_VALUE. {:name "CL_INVALID_VALUE", :code -30, :type :opencl-error, :details nil}
              uncomplicate.clojurecl.utils$error.invokePrim(utils.clj:96)
              uncomplicate.clojurecl.utils$error.invoke(utils.clj:99)
              uncomplicate.clojurecl.info$eval9800$fn__10010.invoke(info.clj:888)
              uncomplicate.clojurecl.info$eval9109$fn__9110$G__9100__9115.invoke(info.clj:247)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253$fn__18254$fn__18255.invoke(core_test.clj:81)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253$fn__18254.invoke(core_test.clj:75)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253.invoke(core_test.clj:75)
              uncomplicate.clojurecl.core_test$eval18252.invoke(core_test.clj:75)

FAIL "Root level devices resource management." at (core_test.clj:82)
    Expected: 1
      Actual: clojure.lang.ExceptionInfo: OpenCL error: CL_INVALID_VALUE. {:name "CL_INVALID_VALUE", :code -30, :type :opencl-error, :details nil}
              uncomplicate.clojurecl.utils$error.invokePrim(utils.clj:96)
              uncomplicate.clojurecl.utils$error.invoke(utils.clj:99)
              uncomplicate.clojurecl.info$eval9800$fn__10010.invoke(info.clj:888)
              uncomplicate.clojurecl.info$eval9109$fn__9110$G__9100__9115.invoke(info.clj:247)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253$fn__18254$fn__18257.invoke(core_test.clj:82)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253$fn__18254.invoke(core_test.clj:75)
              uncomplicate.clojurecl.core_test$eval18252$fn__18253.invoke(core_test.clj:75)
              uncomplicate.clojurecl.core_test$eval18252.invoke(core_test.clj:75)
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000000000000, pid=3400, tid=140586987812608
#
# JRE version: Java(TM) SE Runtime Environment (8.0_45-b14) (build 1.8.0_45-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.45-b02 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  0x0000000000000000
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/ludwik/code/lg-clones/clojurecl/hs_err_pid3400.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

**Comment by LudoTheHUN** [comment]
Tried to run again as root... Same set of errors.

**Comment by blueberry** [comment]
Hi, As you have probably noted, Nvidia gives you support only for Opencl 1.2 (very latest drivers) or 1.1, which is uo to nvidia and not Opencl. To compile ClojureCL and run lein midje (not tlein test) you need all OpenCL 2.0 features, so it obviously will not work on nvidia. However, you can USE the library with any opencl version; follow the Getting started guide. The lib is in clojars. For nvidia, you'll have to set OpenCL version setting for enq-nd method (see the api doc, the link is in the front page).

**Comment by LudoTheHUN** [comment]
Thank you for that, I got it working.

Notes for posterity:

I switched *opencl-2*   to `false` as per the documentation.

Then, avoiding `with-release` (which supposedly touches opencl2 functionality, since it kills the jvm), I was able to run a basic kernel as per the openclinaction/ch04.clj example and read out the result.

```clojure
(ns clocltest.core
  (:require [uncomplicate.clojurecl
             [core :refer :all]
             [info :refer [info endian-little]]]
            [vertigo
             [bytes :refer [direct-buffer byte-seq]]
             [structs :refer [wrap-byte-seq int8]]])
  )

(alter-var-root 
         (var uncomplicate.clojurecl.core/*opencl-2*)
         (fn [f]  false))

(let 
  [dev (first (devices (first (platforms))))
   work-sizes (work-size [1])
   ctx (context [dev])
   cqueue (command-queue ctx dev)
   host-msg (direct-buffer 16)
   cl-msg (cl-buffer ctx 16 :write-only)
   program-source
   "__kernel void hello_kernel(__global char16 *msg) {\n    *msg = (char16)('H', 'e', 'l', 'l', 'o', ' ',
   'k', 'e', 'r', 'n', 'e', 'l', '!', '!', '!', '\\0');\n}\n"
   prog (build-program! (program-with-source ctx [program-source]))
   hello-kernel (kernel prog "hello_kernel")
   read-complete (event)
   ]
  (set-args! hello-kernel cl-msg) 
  (enq-nd! cqueue  hello-kernel work-sizes)
  (enq-read! cqueue cl-msg host-msg read-complete)
  ;;should use core.async to wait for read to finish as per openclinaction/ch04.clj  ,else once could read the buffer before it was written to.
  (apply str (map char
                         (wrap-byte-seq int8 (byte-seq host-msg))))
  )
```

**Comment by blueberry** [comment]
Can you please open a new issue for with-release and provide some minimal test case, since it should not depend on OpenCL2. I suspect it is due to some specificity in the nVidia implementation...

**Comment by LudoTheHUN** [comment]
will do.
